tests/unit/amo/test_installAddon.js

```diff
@@ -220,6 +225,7 @@ describe(__filename, () => {
     sinon.assert.notCalled(_addonManager.getAddon);
   });
 
+  // FIXME
```

Comment:
We can probably skip this one: there are a ton of other tests below that ensure we call `getAddon()` when rendering.